core/templates/dev/head/pages/exploration-player-page/exploration-player-page.mainpage.html

```diff
@@ -40,5 +40,6 @@
     <script src="/templates/dev/head/mathjaxConfig.js"></script>
     <script src="/third_party/static/MathJax-2.7.5/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
     @require('../footer_js_libs.html')
+    @require('../../../../../../extensions/dependencies/dependency_html.html')
```

Comment:
Do you even need the whole "dependency" system any more, now that we are serving all deps for all explorations? It seems to me like you could use webpack to manage the different dependencies (codemirror etc.), especially since some pages need codemirror in a context that does not include interactions (like the topic/skill pages) which was what extensions/dependencies/ was originally for.

If there isn't a "specific dependency" system then perhaps all the dependency stuff should move to the root folder of extensions/interactions.